### ISSUE TITLE
WAT does not seem to work with Godot 3.6.3 on Linux

### ISSUE THREAD

**Issue by isokissa** [AlexDarigan/WAT-GDScript#195]
It does not recognize my tests in `arena.test.gd` script that I have put into `tests` directory, according to instructions:  

```
extends WAT.Test

func test_proba() -> void: 
	asserts.is_true(1 == 1)

```

**Comment by isokissa** [comment]
Hmm, could it be because of case sensitive file names? If you were developing on Windows maybe you did not pay attention to the case used in filenames? I am getting some errors in the command line when running godot, like: 

`ERROR: load: Cannot load source code from file 'res://addons/WAT/ui/scripts/GUI.gd'.
`

And indeed the file is named `gui.gd`, not `GUI.gd`

**Comment by AlexDarigan** [comment]
Both Godot and Git are terrible with cached names so this problem comes up from time to time. Did you change your local copy to the correct filename and check?

**Comment by dmlo** [comment]
It's happening on Linux because the file system is case sensitive. The script is looking for `GUI.gd` but the file is named `gui.gd`. Renaming the file allows WAT to run normally.

No issue on Windows because the file system is not case sensitive.

**Comment by dmlo** [comment]
Renaming the file to `GUI.gd` allows WAT to run under linux.

**Comment by AlexDarigan** [comment]
Should be resolved by 40431c5be2f349376379da75287e92e53cd519cb.
